api/cloudcontroller/ccv2/service_broker.go

```diff
@@ -12,14 +15,23 @@ type ServiceBroker struct {
 	GUID string
 	// Name is the name of the service broker.
 	Name string
+	// URL is the URL of the service broker.
```

Comment:
Make sure GoDoc matches the struct field name.

Also please alphabetize the fields in this struct.

Reply:
Good catch, thanks 👍